src/components/menu/Menu.tsx

```diff
@@ -11,37 +11,49 @@ export type TMenuItem = {
   title: string;
   route: string | null;
   action: string | null;
-  titleAdditionIfModifierTrue?: string;
-  titleAdditionIfModifierFalse?: string;
+  isWithModifier?: boolean;
+  isMobileOnly?: boolean;
 }
 
 interface IMenu {
   menuItems?: TMenuItem[];
   isShown?: boolean;
   isWithTitle?: boolean;
   className?: string;
-  modifier?: boolean;
+  modifier: { [key: string]: boolean }
+  isTest?: boolean;
   handleAction?: (action: string) => void;
 }
 
 const Menu: FC<IMenu> = ({
   menuItems = MENU_ITEMS,
   isShown = true,
   isWithTitle = true,
-  modifier = true,
+  isTest = false,
+  modifier,
   className,
   handleAction
 }): ReactElement => {
+  let isMobile = false;
+  if (!isTest && window) {
```

Comment:
❗️
```suggestion
  let isMobile = false;
  
  if (!isTest && window) {
```

В идеале это отдельным хуком сделать

Reply:
вообще все эти отступы, наверное в идеале, настраиваются в преттире/еслинте ?